app/config/config.yml

```diff
@@ -12,11 +12,17 @@ framework:
     form:            true
     csrf_protection: true
     validation:      { enable_annotations: true }
-    templating:      { engines: ['twig'], assets_version: 3 }
+    templating:
+        engines: ['twig']
+    assets:
+        version: 3
     default_locale:  "%locale%"
     trusted_proxies: ~
-    trusted_hosts: ['.*\.?afsy.fr$']
-    session:         ~
+    trusted_hosts: ['.*\.?afsy.fr$', 'localhost']
```

Comment:
I know it's not the scope of your PR but `.*` is not required right?

Reply:
This should be only in dev env. Mybad! Indeed I don't think `.*` is necessary.